articles/flask-database-integration-with-sqlalchemy/index.md

```diff
@@ -0,0 +1,329 @@
+
+In the process of creating models, we find that some models do have some relationship to one another or with other data, e.g a user might have a specific ID, such that this specific ID also refers back to the user, but how do we tell the database about the existence of this symbiotic relationship? To do this, we first need to understand the basic database relationship and how they are being initialized in our models.
+
+## Database Relationship
+In the process of working with databases, the following are the two types of relationship that you would frequently interact with when working with tables:
+- One to Many
+- One to One
+
+In our understanding of the various database relationships, consider the creation of a Pet management system. The application will consist of the following models: PetHandler, PetCategory, PetComment and the implementation of this model is as follows:
+```python
+class PetHandler(db.Model):
+   handler_id = db.Column(db.Integer, primary_key = True)
+   handler_name = db.Column(db.String(150), nullable = False)
+ 
+class PetCategory(db.Modle):
+   pet_id = db.Column(db.Integer, primary_key = True)
+   pet_name = db.Column(db.String(100), nullable = False)
+   pet_category = db.Column(db.String(50), nullable = False )
+ 
+class PetComnment(db.Model):
+   comment_id = db.Column(db.Integer, primary_key = True)
+   comment_content = db.Column(db.String(200), nullable = False)
+```
+You might not understand the implementation of the above code, but focus on the concept of database relationship, we will get to the point of you understanding the components of creating a database.
+
+## One-to-Many Relationship
+From our Pet management system as an analogy, an example of a one-to-many relationship is analyzed between a Pet Handler and a Pet in the Pet Category, such that a pet handler can be assigned to multiple (many) pets to care for. 
+
+The representation of this relationship in our model is by following this process:
+
+We create a column containing a ForeignKey() in the PetCategory class. Simply, the pet handler is the attribute having multiple relationships with the pets, therefore the pets will have a column consisting of a  ForeignKey() to the pet handler's unique identify key which is the pet handler’s id.
+
+```python
+class PetCategory(db.Model):
+    pet_id = db.Column(db.Integer, primary_key = True)
+    pet_name = db.Column(db.String(100), nullable = False)
+    pet_category = db.Column(db.String(50), nullable = False )
+    handler_id = db.Column(db.Integer, db.ForeignKey('pethandler.handler_id'), nullable = False)
+```
+
+The last line of code, with the handler_id variable name, initializes the existence of a one-to-many relationship from the pet handler to the pets. The db.ForeignKey receives an argument pointing to the handler_id column in the pethandler table. Note that the pethandler class name has to be in lowercase which tells SQLAlchemy that we are referring to a table.
```

Comment:
```suggestion
The last line of code, with the `handler_id` variable name, initializes the existence of a one-to-many relationship from the pet handler to the pets. The `db.ForeignKey` receives an argument pointing to the `handler_id` column in the pethandler table. Note that the pethandler class name has to be in lowercase which tells SQLAlchemy that we are referring to a table.
```
For any keyword/variable that you're using in code snippet, just enclose them within ` `` `

Reply:
Make sure that you make these changes throughout the article

Reply:
@corpsgeek